MobSF/views/api/rest_api.py

```diff
@@ -198,3 +201,25 @@ def api_view_source(request):
     else:
         response = make_api_response({"error": "Missing Parameters"}, 422)
     return response
+
+
+@request_method(['POST'])
+@csrf_exempt
+def api_manifest(request):
+    """
+    :param type ipa/apk
+    """
+    params = ['bin', 'type', 'hash']
+    if set(request.POST) >= set(params):
+        scan_type = request.POST.get('type')
+        md5 = request.POST.get('hash')
+        if scan_type in ['eclipse', 'studio', 'apk']:
+            resp = manifest_view.run(request, api=True)
+        elif scan_type == 'ipa':

```

Comment:
How about `ios source code`.  Does this cover manifest file view for ios source code zip uploads?